src/sql/base/parts/editableDropdown/browser/dropdown.ts

```diff
@@ -361,7 +357,7 @@ export class Dropdown extends Disposable implements IListVirtualDelegate<string>
 	}
 
 	private _inputValidator(value: string): IMessage | null {
-		if (!this._input.hasFocus() && !this._selectList.isDOMFocused() && this._dataSource.values && !this._dataSource.values.some(i => i === value)) {
+		if (!this._input.hasFocus() && this._input.isEnabled() && !this._selectList.isDOMFocused() && !this._dataSource.values.some(i => i === value)) {
```

Comment:
I did also update it so that we don't do the validation if the input is disabled - which to me seems correct since if it's disabled the user can't actually do anything about it anyways.